packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/nutrition.md

```diff
@@ -0,0 +1,14 @@
+This guide explains how to implement an Authorization Code flow for your app with Okta.
+
+---
+
+**Learning outcomes**
+
+* Understand the OAuth 2.0 Authorization Code flow.
+* Set up your app with the Authorization Code grant type.
+* Implement the Authorization Code flow in Okta.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
```

Comment:
We could remove the URL in the link, and replace with (/signup) -- as in the Note that comes after.

Reply:
I thought we used the full URL for the /signup page to open a new tab? I'll confirm. Thanks

Reply:
I spoke with Susan about this convention: most of our /signup links are full URLs to open a new tab (with the exception of some quick-start guides and the ApiAmProdWarning snippet note). I'll follow the full URL convention for now. Thanks for pointing this out :-)

Reply:
No problem with this. I like opening the new tab as well.